docs/PREACT_DISPLAYING_DATA.md

```diff
@@ -267,6 +279,151 @@ class Result extends Component {
 }
 ```
 
+
+### Variants
+When variants are enabled, each result (in controller.store.results) contains a variants object with the following properties:
+
+`variants.active` - The currently selected variant's data.
+
+`variants.data` - Array of all variant data.
+
+`variants.selections` - Contains variant selection data (see below).
+
+
+and the following functions:
+
+`variants.setActive` - Function to update the selected variant.
+
+`variants.update` - Updates the variant data (see below for more details).
+
+`variants.makeSelections` - For internal usage only.
+
+`variants.refineSelections` -  Updates available variant options based on current selections to prevent invalid combinations
+
+
+To enable variants:
+
+1. Get variant data from your platform. See supported platform groovy scripts [here](https://searchspring.github.io/snap/#/package-platforms).
```

Comment:
I'm not sure we want to mention "groovy scripts" in our Snap docs - this takes this documentation into indexing and SMC configuration and I don't think we want to go there.

Perhaps we should just remove this step.